gows.yml

```diff
@@ -0,0 +1 @@
+package_name: github.com/KTamas-Bitrise/steps-gradle-unit-test
```

Comment:
please gitignore

Reply:
no, rather set it to the upstream package name (`github.com/bitrise-io/steps-gradle-unit-test`)

Reply:
@viktorbenei @trapacska done